modules/liquidity-pools/base/src/lib.rs

```diff
@@ -21,7 +21,7 @@ mod tests;
 
 pub trait Trait<I: Instance = DefaultInstance>: system::Trait {
 	type Event: From<Event<Self, I>> + Into<<Self as frame_system::Trait>::Event>;
-	type LiquidityCurrency: BasicCurrency<Self::AccountId, Balance = Balance>;
+	type LiquidityCurrency: BasicCurrency<Self::AccountId, Balance = Balance> + BasicReservableCurrency<Self::AccountId>;
```

Comment:
`LiquidityCurrency` is `AUSD`, but I think we should reserve `LAMI` instead? So you will need to add a new `type DepositCurrency: BasicReservableCurrency<Self::AccountId>`